projects/storefrontapp-e2e-cypress/cypress/integration/regression/notification/stock-notification.core-e2e-spec.ts

```diff
@@ -3,7 +3,7 @@ import { viewportContext } from '../../../helpers/viewport-context';
 import { clearAllStorage } from '../../../support/utils/clear-all-storage';
 
 describe('My interests', () => {
-  viewportContext(['mobile', 'desktop'], () => {
+  viewportContext(['desktop'], () => {
```

Comment:
It was already changed in another e2e refactor, please update your branch with the current develop. This test is now for both mobile and desktop views.

Reply:
Yes but looking at other core tests, they should only be running in desktop mode. 

I created a stock-notification.e2e-spec (non-core) that runs both mobile and desktop view. Then the core file only runs the desktop view. Let me know what you think of this